com.smallstep.Smallstep.yml

```diff
@@ -0,0 +1,101 @@
+id: com.smallstep.Smallstep
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: smallstep
+rename-icon: smallstep
+
+finish-args:
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf # For GNOME proxy resolution
+  - --env=GIO_EXTRA_MODULES=/app/lib/gio/modules # For GNOME proxy resolution
+  - --env=GSETTINGS_BACKEND=dconf # For GNOME proxy resolution
+  - --filesystem=~/.config/dconf:ro # For GNOME proxy resolution
+  - --filesystem=~/.config/kioslaverc # For KDE proxy resolution (KDE5 only)
+  - --filesystem=xdg-run/dconf # For GNOME proxy resolution
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=system-bus
+  - --socket=session-bus
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --device=all
+
+modules:
+  # systray/menu support
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  # TODO(josh) - required, wails depends in webkit2gtk-4.0, switch to Gnome runtime
+  # once wails supports 4.1
+  - name: webkit2gtk-4.0
```

Comment:
I removed the majority of the nested modules here and updated the compiler flags to ensure to narrow this to just what we need.